--- kde-i18n.org/pt_BR/docs/kdebase/kcontrol/arts/index.docbook.org	2005-04-25 21:58:56.344676600 +0200
+++ kde-i18n/pt_BR/docs/kdebase/kcontrol/arts/index.docbook	2005-04-25 21:59:09.471680992 +0200
@@ -86,7 +86,7 @@
 <para
 >Você pode saber mais sobre o &arts; digitando <userinput
 >help:/artsbuilder</userinput
-> na barra de localização do &konqueror;,  ou procurando na documentação do &artsbuilder; no &khelpcenter;. </para>
+> na barra de localização do &konqueror;,  ou procurando na documentação do &arts-builder; no &khelpcenter;. </para>
 </tip>
 
 <para
