diff -ur kde-i18n-3.2.0/es/docs/quanta/quanta.orig/debugging-quanta.docbook kde-i18n-3.2.0/es/docs/quanta/quanta/debugging-quanta.docbook
--- kde-i18n-3.2.0/es/docs/quanta/quanta.orig/debugging-quanta.docbook	2004-01-28 12:19:05.290733952 +0100
+++ kde-i18n-3.2.0/es/docs/quanta/quanta/debugging-quanta.docbook	2004-01-28 12:26:32.605731736 +0100
@@ -1,4 +1,4 @@
-<?xml version="1.0" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 
 <sect1 id="debugging">
 <title
Tylko w kde-i18n-3.2.0/es/docs/quanta/quanta: debugging-quanta.docbook~
Tylko w kde-i18n-3.2.0/es/docs/quanta/quanta: index.cache.bz2
diff -ur kde-i18n-3.2.0/es/docs/quanta/quanta.orig/index.docbook kde-i18n-3.2.0/es/docs/quanta/quanta/index.docbook
--- kde-i18n-3.2.0/es/docs/quanta/quanta.orig/index.docbook	2004-01-28 12:19:05.291733800 +0100
+++ kde-i18n-3.2.0/es/docs/quanta/quanta/index.docbook	2004-01-28 12:25:54.661500136 +0100
@@ -1,4 +1,4 @@
-<?xml version="1.0" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1.2-Based Variant V1.1//EN"
 "dtd/kdex.dtd" [
   <!ENTITY kappname "&quanta;">
Tylko w kde-i18n-3.2.0/es/docs/quanta/quanta: index.docbook~
