--- kde-i18n.org/docs/kdewebdev/kommander/index.docbook.org	2005-06-04 23:39:07.115390336 +0000
+++ kde-i18n/docs/kdewebdev/kommander/index.docbook	2005-06-04 23:40:15.235034576 +0000
@@ -107,9 +107,6 @@
 >A Equipe de Desenvolvimento do &kommander;</holder>
 </copyright>
 
-<legalnotice
->&FDLNotice;</legalnotice>
-
 <!-- Date and version information of the documentation
 Don't forget to include this last date and this last revision number, we
 need them for translation coordination !
@@ -156,4 +153,4 @@
 &installation;
 &glossary;
 
-</book>
\ No newline at end of file
+</book>
