diff -ur kde-i18n-es-3.5.1.org/docs/common/Makefile.in kde-i18n-es-3.5.1/docs/common/Makefile.in
--- kde-i18n-es-3.5.1.org/docs/common/Makefile.in	2006-01-21 19:03:57.000000000 +0000
+++ kde-i18n-es-3.5.1/docs/common/Makefile.in	2006-01-23 07:38:34.391130937 +0000
@@ -512,6 +512,7 @@
 install-nls:
 	$(mkinstalldirs) $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/common
 	@for base in lgpl-translated.html gpl-translated.html fdl-translated.html kde-localised.css logotp3.png ; do \
+	  rm -f $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/common/$$base ;\
 	  echo $(INSTALL_DATA) $$base $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/common/$$base ;\
 	  $(INSTALL_DATA) $(srcdir)/$$base $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/common/$$base ;\
 	done
